src/frontend/cfg_parser.mli

```diff
@@ -69,7 +69,8 @@ val edge_list_of_stmt_list :
   exit:Cfg.Loc.t ->
   ret:Cfg.Loc.t ->
   exc:Cfg.Loc.t ->
-  ?brk:Cfg.Loc.t option ->
+  ?brk:Cfg.Loc.t option * (CST.void_type * Cfg.Loc.t) list ->
```

Comment:
Related to my other comment, I think this should be a `Cfg.Loc.t option * string Cfg.Loc.Map.t`.

i.e. instead of carrying around a `CST.void_type` (i.e. a token and some source location info) when we're only going to use the contents of the token, just keep the contents of the token.
Going with a hashmap `'a Cfg.Loc.Map.t` instead of an association list`('a * Cfg.Loc.t) list` doesn't really matter for performance (since these are never going to have more than a handful of elements) but does make the intended meaning a bit clearer IMO

Reply:
I think this might also be getting to the point where we should just pack all of these named location arguments up in a `loc_ctx` record... that's not directly related to this PR, though, just putting that thought down here.

Reply:
Sorry, make that `Loc.t String.Map.t`, not `string Loc.Map.t` -- we want to map strings (i.e. java labels) to locations, not the other way around.